modules/boost.exception/1.83.0/presubmit.yml

```diff
@@ -0,0 +1,42 @@
+matrix:
+  platform:
+    - debian10
+    - debian11
+    - macos
+    - macos_arm64
+    - ubuntu2004
+    - ubuntu2204
+    - windows
+  bazel: [7.x]
+tasks:
+  verify_targets:
+    name: Verify build targets
+    platform: ${{ platform }}
+    bazel: ${{ bazel }}
+    build_flags:
+      - '--process_headers_in_dependencies'
+    build_targets:
+      - '@boost.exception//:boost.exception'
+bcr_test_module:
+  module_path: test
+  matrix:
```

Comment:
Not sure if I can change this to:
matrix:
  platform:
    - debian10
    - debian11
    - macos
    - macos_arm64
    - ubuntu2004
    - ubuntu2204
    - windows
  bazel: [7.x]

tasks:
  verify_targets:
    name: Verify build targets
    platform: ${{ matrix.platform }}
    bazel: ${{ matrix.bazel }}
    build_flags:
      - '--process_headers_in_dependencies'
    build_targets:
      - '@boost.exception//:boost.exception'

bcr_test_module:
  module_path: test
  tasks:
    run_test_module:
      name: Run test module
      platform: ${{ matrix.platform }}
      bazel: ${{ matrix.bazel }}
      build_flags:
        - '--process_headers_in_dependencies'
      build_targets:
        - //...
      test_targets:
        - //...

Reply:
I _think_ you can just indent everything after the first `tasks:` line (inclusive)?